jenkins/osa_onmetal.functions.groovy

```diff
@@ -368,6 +368,36 @@ def stop_api_uptime_tests() {
     """
 }
 
+def setup_parse_persistent_resources(){
+    
+    String host_ip = get_onmetal_ip()
+    
+    sh """
+    ssh -o StrictHostKeyChecking=no root@${host_ip} '''
+    git clone https://github.com/osic/persistent-resources-tests-parse.git
+    pip install -e /root/persistent-resources-tests-parse
```

Comment:
Why are you installing this repo in development mode? At this point this should be an official repo.